### PR TITLE
Add unique validation messages for Sketchmons

### PR DESCRIPTION
**Pull request by MacChaeger**
I left this to sit overnight so I could review it with fresh eyes in the morning and forgot about it for 12 days... Anyway, this should give users a much more informative message than "(Pokemon) cannot learn (move)" when validating a Sketchmons set that either has a banned sketch move or more than one.

### CODE REVIEW

File: team-validator.js [hunk 1]
```diff
@@ -310,6 +311,14 @@ class Validator {
 						} else {
 							problemString = problemString.concat(`.`);
 						}
+						// Sketchmons hack part 2
+						if (allowOneSketch) {
+							if (noSketch.indexOf(move.name) >= 0) {
+								problemString = problemString.concat(` (${move.name} is banned from being sketched in ${format.name}.)`);
```
Comment (Zarel) [inline team-validator.js]: Is this going to cause a problem even if the move is legally obtainable by other methods?

File: team-validator.js [hunk 2]
```diff
@@ -310,6 +311,14 @@ class Validator {
...
+							} else if (set.sketchmonsMove !== move.id) {
+								problemString = problemString.concat(` (You cannot sketch more than one move per Pok\u00E9mon.)`);
```
Comment (Zarel) [inline team-validator.js]: Ditto this.

File: team-validator.js (hunk 1 above)
Comment (MacChaeger) [reply team-validator.js]: If we get this far, it means the move isn't normally legal.

Comment (Zarel) [reply team-validator.js]: Oh, I see now.

**Comment by TheImmortal** [comment]
Sketchmons is being removed tomorrow so this is of no use.

**Comment by MacChaeger** [comment]
rip

**Pull request closed without merging**